gammapy/image/models/core.py

```diff
@@ -110,6 +111,85 @@ def evaluate(lon, lat, lon_0, lat_0, sigma):
         return norm * np.exp(exponent)
 
 
+class SkyElongatedGaussian(SkySpatialModel):
+    r"""Two-dimensional elongated Gaussian model (defined on the plane).
+
+    .. math::
+
+             f(\text{lon}, \text{lat}) = \frac{1}{2\pi\sigma_{\text{lon}}\sigma_{\text{lat}}} e^{-a\left(\text{lon} - \text{lon}_{0}\right)^{2}  -b\left(\text{lon} - \text{lon}_{0}\right)\left(\text{lat} - \text{lat}\right)  -c\left(\text{lat} - \text{lat}_{0}\right)^{2}}
+
+    Using the following definitions:
+
+            a = \left(\frac{\cos^{2}{\left (\theta \right )}}{2 \sigma_{\text{lon}}^{2}} +
+            \frac{\sin^{2}{\left (\theta \right )}}{2 \sigma_{\text{lat}}^{2}}\right)
+
+            b = \left(\frac{\sin{\left (2 \theta \right )}}{2 \sigma_{\text{lon}}^{2}} -
+            \frac{\sin{\left (2 \theta \right )}}{2 \sigma_{\text{lat}}^{2}}\right)
+
+            c = \left(\frac{\sin^{2}{\left (\theta \right )}}{2 \sigma_{\text{lon}}^{2}} +
+            \frac{\cos^{2}{\left (\theta \right )}}{2 \sigma_{\text{lat}}^{2}}\right)
+
+    where :math:`\sigma_{\text{lon}}` and :math:`\sigma_{\text{lat}}` are the 68% containment semi-axes of the gaussian before rotation, and :math:`\theta` is the rotation angle (with respect to the constant latitude axis).
+
+    Parameters
+    ----------
+    lon_0 : `~astropy.coordinates.Longitude`
+        :math:`lon_0`
+    lat_0 : `~astropy.coordinates.Latitude`
+        :math:`lat_0`
+    sigma_lon : `~astropy.coordinates.Angle`
+        :math:`\sigma_{\text{lon}}`
+    sigma_lat : `~astropy.coordinates.Angle`
+        :math:`\sigma_{\text{lat}}`
+    theta : `~astropy.coordinates.Angle`
+        :math:`\theta}`
+    """
+
+    def __init__(self, lon_0, lat_0, sigma_lon, sigma_lat, theta):
+        self.parameters = Parameters(
+            [
+                Parameter("lon_0", Longitude(lon_0)),
+                Parameter("lat_0", Latitude(lat_0)),
+                Parameter("sigma_lon", Angle(sigma_lon), min=0),
+                Parameter("sigma_lat", Angle(sigma_lat), min=0),
+                Parameter("theta", Angle(theta)),
```

Comment:
We might want to restrict the fit range for the angle by default to (0 < theta < 90) or (0 < theta < pi/2) to prevent degenerate fit solutions. This should be also added to the docstring...